test/index.html

```diff
@@ -5,11 +5,12 @@
 <link rel="stylesheet" href="../build/test.css">
 <style>
   .fixture-Box {
-    background: rgb(100, 150, 50);
-    background: rgb(100, 150, 50);
-    border: 1px solid black;
+    background: rgba(100, 150, 50, .75);
+    border: 1px solid #888;
+    font-family: sans-serif;
     margin-bottom: 10px;
```

Comment:
We should remove this margin it can be misleading.
People may make the assumption that wrapping cells with gutters also get vertical spacing.

By the way, maybe we should also add vertical spacing to `Grid--withGutters`.